pages/explore.tsx

```diff
@@ -84,24 +84,31 @@ class Search extends React.Component<{ router: NextRouter, wpData: WPAtlas[] },
     }
 
     @action.bound
-    setFilter(groupNames: string[], actionMeta: ActionMeta<ExploreSelectedFilter>) {
+    setFilter(actionMeta: ActionMeta<ExploreSelectedFilter>) {
         //const filters = Object.assign({}, this.state.filters);
 
         let newFilters:ExploreSelectedFilter[] = this.selectedFilters;
         if (actionMeta && actionMeta.option) {
-            // first remove the item
-            newFilters = this.selectedFilters.filter((o)=>{
-                return o.group !== actionMeta!.option!.group! || o.value !== actionMeta!.option!.value!;
-            });
-
-            if (actionMeta.action === 'select-option') {
-                const option = actionMeta.option;
-                newFilters = newFilters.concat([option]);
+            switch (actionMeta.action) {
+                case 'clear':
```

Comment:
should we define an enum for these actions? `FilterAction.Clear` or something like that

Reply:
good point. done